xbmc/guilib/GUIWindowManager.cpp

```diff
@@ -779,6 +780,16 @@ void CGUIWindowManager::ActivateWindow_Internal(int iWindowID, const std::vector
     return;
   }
 
+  if (iWindowID == WINDOW_VIDEO_NAV)
+  {
+     if (params.front().substr(0, 4) != "root" && params.front().substr(0, 10) != "library://")
+     {
+       CLog::Log(LOGDEBUG, "****************** not entering : %s", params.front());
+//     if !g_passwordManager.IsItemUnlocked( ***** , strLockType))
```

Comment:
If this code is not required remove it.

Reply:
this code is required, as we need to check if the current item is unlocked.
but i'm stuck here because g_passwordManager.IsItemUnlocked expects a CFileItem.
i must admit i have no clue how to resolve the path (which is passed by params.front()) to a CFileItem (the corresponding mediasource), so we can check. don't know if this is possible at all.
any help on this by experienced team-members is appreciated